tests/utils.py

```diff
@@ -176,7 +176,7 @@ def compare_two_settings(model: str,
         env2: The second set of environment variables to pass to the API server.
     """
 
-    tokenizer = AutoTokenizer.from_pretrained(model)
+    tokenizer = AutoTokenizer.from_pretrained(model, trust_remote_code=True)
```

Comment:
It would be better to make this an optional setting instead of enabling `trust_remote_code` by default.